packages/ckeditor5-engine/src/view/domconverter.js

```diff
@@ -237,8 +238,9 @@ export default class DomConverter {
 					this.bindDocumentFragments( domElement, viewNode );
 				}
 			} else if ( viewNode.is( 'uiElement' ) ) {
-				// UIElement has its own render() method (see #799).
```

Comment:
I guess it's still true for elements other than `$comment`

Reply:
Rewritten a bit to restore this deleted comment.